src/app/components/DataLoader/DataLoader.jsx

```diff
@@ -21,4 +22,4 @@ DataLoader.contextTypes = {
   router: PropTypes.object,
 };
 
-export default DataLoader;
+export default connect()(DataLoader);
```

Comment:
`connect` is not needed here

Reply:
I think it is needed to make `dispatch` available in the props?

Reply:
Got it. It is possible to get `dispatch` by importing the store to `dataLoaderUtil`. But for some reason, that does not work for no-JS